.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,61 @@
+name: docker-ci
+
+on:
+  push:
+    branches:
+      - '*'
+    #  - main
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  # enable manual trigger
+  workflow_dispatch:
+
+env:
+  DOCKER_REPOSITORY: mentat-is
+  DOCKER_IMAGE_NAME: gulp
```

Comment:
do you have a valid docker hub account?

Reply:
We do! The repository and image name should be `mentatis/gulp-core` (https://hub.docker.com/r/mentatis/gulp-core)

Reply:
awesome, i will update it later then